spec/inverted-index-spec.js

```diff
@@ -1,74 +1,96 @@
 const InvertedIndexhelper = require('./inverted-index-helper');
-const IndexKlass = require('./inverted-index');
+const IndexClass = require('./inverted-index');
+const empty = require('../spec/empty');
+const invalidFile = require('../spec/invalid.json');
 const fs = require('fs');
 const path = require('path');
 
 const book = fs.readFileSync(path.resolve(__dirname, 'books.json'));
 
-
 describe('Inverted Index', () => {
-  const helpers = new InvertedIndexhelper();
-  const InvertedIndex = new IndexKlass(InvertedIndexhelper);
+  const InvertedIndex = new IndexClass(InvertedIndexhelper);
   const file1 = book;
-  const emptyFile = [];
-  const notJson = 'books.js';
-  InvertedIndex.files.books = JSON.parse(book);
-  InvertedIndex.createIndex('books');
+  const file2 = empty;
+  const file3 = invalidFile;
+
   afterEach(() => {
     InvertedIndex.indexTable = {};
   });
-
-
+  
   it('should be truthy for the instance of the class', () => {
-    expect(InvertedIndex instanceof IndexKlass).toBeTruthy();
+    const indexInstance = InvertedIndex;
+    expect(InvertedIndex instanceof IndexClass).toBeTruthy();
   });
-
   it('should return zero for the length of the indexes', () => {
-    expect(Object.keys(InvertedIndex.getIndex).length).toBe(0);
+    const indexInstance = InvertedIndex;
+    expect(Object.keys(InvertedIndex.indexTable).length).toBe(0);
```

Comment:
### Critical

You shouldn't be accessing instance properties directly, your `getIndex` method should be returning the `indexTable` instead.